### ISSUE TITLE
Attach nav link event to `<a>` instead of `<li>`

### ISSUE THREAD

**Issue by danieltott** [code-newbies/codeland#61]
Currently the javascript that closes the nav menu when a user clicks on an item is attached to the nav `li` elements:

https://github.com/code-newbies/codeland/blob/98a5c7cbc1ac123dd707062199ea672d219089a8/source/javascripts/main.js#L6-L8

`listitem` elements are not focusable by default - there's ways to work around this, but a much easier approach would be to simply attach that behavior to the `<a>` elements.

![image](https://user-images.githubusercontent.com/360261/128742572-a46751c4-1e80-48d5-84af-a14306ce5eeb.png)
